docs/finetune.md

```diff
@@ -46,21 +46,30 @@ Training:
     GPU: 1
   accelerate_mode: GPU_DDP
 ```
+For HPU, set `device` to HPU, set HPU number in `resources_per_worker` and set `accelerate_mode` to HPU_DDP or HPU_DEEPSPEED.
+```
+Training:
+  device: HPU
+  resources_per_worker:
+    CPU: 1
+    HPU: 1
+  accelerate_mode: HPU_DDP
+```
 Please see [finetune_parameters.md](finetune_parameters.md) for details of the parameters.
 
 
 ### Validated Models
 The following models have been verified on Intel CPUs or GPUs.
-|Model|CPU|GPU|
-|---|---|---|
-|llama/llama2|✅|✅|
-|mistral|✅|✅|
-|pythia|✅|✅|
-|gpt-j|✅|✅|
-|bloom|✅|✅|
-|opt|✅|✅|
-|mpt|✅||
-|gpt2|✅|✅|
+|Model|CPU|GPU|HPU|
```

Comment:
Also update the description above, "The following models have been verified on Intel CPUs,  GPUs and HPUs."

Reply:
Please correct this one.